postgres/server/init.sls

```diff
@@ -107,6 +107,10 @@ postgresql-conf:
 {%- endif %}
 
 {%- set pg_hba_path = salt['file.join'](postgres.conf_dir, 'pg_hba.conf') %}
+{%- set acls = postgres.acls %}
+{%- if postgres.version|to_num >= 10 %}
```

Comment:
Please use `int` instead of `to_num` (salt v2018), or even `str_to_num` (salt v2017). In this case the Jinja [builtin filters](http://jinja.pocoo.org/docs/2.10/templates/#builtin-filters) are more portable. Not everybody uses latest Salt versions in production.